### ISSUE TITLE
How to add support to accessibility? 

### ISSUE THREAD

**Issue by vinicius5581** [ahmadsoe/ember-highcharts#169]
I was hopping to add accessibility to my chart but that requires a module as per https://www.highcharts.com/docs/accessibility/accessibility-module. 

Thoughts anyone?

**Comment by nlfurniss** [comment]
@vinicius5581 you should be able to add `accessibility ` to the `includeModules` array [here](https://github.com/ahmadsoe/ember-highcharts#configuration)
